packages/cozy-device-helper/src/flagship.ts

```diff
@@ -0,0 +1,22 @@
+import { Platform } from 'react-native'
+
+export enum FlagshipRoutes {
+  Home = 'home',
+  Cozyapp = 'cozyapp',
+  Authenticate = 'authenticate',
+  Onboarding = 'onboarding',
+  Stack = 'stack'
+}
+
+export interface FlagshipMetadata {
+  immersive?: boolean
+  navbarHeight?: number
+  platform?: typeof Platform
+  route?: FlagshipRoutes
+  statusBarHeight?: number
+  version?: string
+}
+
+export const flagshipMetadata = window.cozy?.flagship || {}
+
+export const isFlagshipApp = (): boolean => window.cozy?.flagship !== undefined
```

Comment:
don't we test this feature?

Reply:
To be honest, I don't think it's worth it, I'm not even sure what to test except the implementation. For me, TypeScript replaces testing here. It ensures I return a boolean, and it ensures my code will not throw.

Reply:
i disagree. If a developer does a `window.cozy?.flagship === undefined` by mistakes, nothing will stop the stop, except a good review.

Reply:
anyway it's a nit